cmd/kubeapps-apis/core/plugins/v1alpha1/plugins_test.go

```diff
@@ -423,10 +423,9 @@ func TestCreateConfigGetterWithParams(t *testing.T) {
 			}
 
 			if tc.expectedErrMsg == nil {
-				if restConfig == nil {
+				if restConfig == nil && restConfig.Host != "" {
```

Comment:
I cannot find a case in which this expression would evaluate to true?
Maybe the `restConfig.Host != ""` needs a separate `if` block to wrap the `got, want :=....`.